File: CPlusPlus/Facade/src/Facade.cpp
/*************************************************************************
    > File Name: Facade.cpp
    > Author: ma6174
    > Mail: ma6174@163.com 
    > Created Time: 2017年03月14日 星期二 10时51分19秒
 ************************************************************************/

#include"Facade.h"

void Facade::Build()
{
	Preprocesser::Preprocess();
	Compiler::Compile();
	Linker::Link();
}

File: Python/Facade/Facade.py
# -*- coding:utf-8 -*-

from Preprocesser import *
from Compiler import *
from Linker import *

class Facade:

    def Build(self):
        Preprocesser.Preprocess()
        Compiler.Compile()
        Linker.Link()

if "__main__" == __name__:
    facader = Facade()
    facader.Build()
        
